MaxText/layers.py

```diff
@@ -225,12 +226,16 @@ def __call__(self, inputs: Array) -> Array:
     if not cfg.int8_training:
       return lax.dot_general(inputs, kernel, ((axis, contract_ind), ((), ())))
     else:
-      aqt_cfg = maxtext_sweeps.sweep1(cfg.fwd_int8, cfg.bwd_int8)
-      aqt_dot_general = aqt.make_dot_general(aqt_cfg)
       aqt_key = self.make_rng('aqt')
-      context = aqt.Context(key=aqt_key, train_step=None)
+      if cfg.use_dqdg:
+        aqt_dq_dg = aqt_dq.make_aqt_dq_dg()
+        return aqt_dq_dg(aqt_key, inputs, kernel, ((axis, contract_ind), ((), ())))
```

Comment:
how come it does not need rng?

Reply:
It needs. Rng is the first argument of this function (not in Context). Whereas rng for aqt v2 is wrapped in a context object.

Reply:
I'm blind :)